### PR TITLE
MNTR-12 data for transaction model first commit

### PR DESCRIPTION
**Pull request by Gokselagan**

### CODE REVIEW

File: src/types/Transaction.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  description?: string;
+  attachments?: Array<{
+    attachmentType:
+      | AttachmentType.Camera
+      | AttachmentType.Document
+      | AttachmentType.Image;
+    url: string;
+  }>;
+  date: string | Date;
+  accountType: AccountType.Personal;
```
Comment (annasvst) [inline src/types/Transaction.tsx:37]:
```suggestion
  accountType: AccountType;
```

File: src/types/Transaction.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  JPY = 'JPY',
+  RUB = 'RUB',
+  EUR = 'EUR',
+  WON = 'WON',
+}
+
+export interface Transaction {
+  category: Category;
+  description?: string;
+  attachments?: Array<{
```
Comment (annasvst) [inline src/types/Transaction.tsx:29]: It's better to type attachments like this: `attachments: Attachment[];`
where 

```
interface Attachment {
  type: AttachmentType;
  url: string;
}
```

Comment (Gokselagan) [reply src/types/Transaction.tsx:29]: Fixed

**Review by annasvst — APPROVED** [review]
LGTM

**Pull request merged**